src/app/components/books/books.scss

```diff
@@ -0,0 +1,49 @@
+$yellow-star: #EAC435; 
+
+.glyphicon {
```

Comment:
For some reason I can't get glyphicon icons to show in the browser. I can see them when I check the html in the console, but not in the normal view. I tried including the bootstrap font and glyphincon libraries in vendorCss, but it did not work. I thought I'd try to replace the glyphicon symbols with font awesome and the stars showed.